### PR TITLE
:globe_with_meridians: Update French Translation

### PR DESCRIPTION
**Pull request by Nsbx**

### CODE REVIEW

File: main/src/assets/lang/fr_FR.json [hunk 1]
```diff
@@ -325,48 +325,48 @@
...
-  "{appName} is the latest version!": "{appName} is the latest version!",
-  "{name} has been updated to the latest version!": "{name} has been updated to the latest version!",
-  "Unable to update at the moment!": "Unable to update at the moment!",
-  "Updating": "Updating",
+  "Click here": "Cliquez ici",
+  "Installing app": "Installation de l'application",
+  "Installation completed": "Installation terminée",
+  "Starting installation": "Début de l'installation",
+  "New": "Nouveau",
+  "NEW": "NOUVEAU",
```
Comment (Nsbx) [inline main/src/assets/lang/fr_FR.json:344]: Duplicate ?
We should not use only one key and make it to uppercase when needed ?

File: main/src/assets/lang/fr_FR.json [hunk 2]
```diff
@@ -325,48 +325,48 @@
...
+  "Check then update": "Vérifier puis mettre à jour",
```
Comment (Nsbx) [inline main/src/assets/lang/fr_FR.json:345]: Duplicate with line 354 ?

File: main/src/assets/lang/fr_FR.json [hunk 3]
```diff
@@ -325,48 +325,48 @@
...
+  "APP may not be available": "L'application peut ne pas être disponible",
+  "Please": "S'il vous plaît",
+  "{appName} is the latest version!": "{appName} est la dernière version !",
+  "{name} has been updated to the latest version!": "{name} a été mis à jour vers la dernière version !",
+  "Unable to update at the moment!": "Impossible de mettre à jour pour le moment !",
+  "Updating": "Mise à jour en cours",
   "Uninstalling": "Désinstallation",
-  "CheckThenUpdate": "Check then update",
-  "Not compatible with {arch} devices.": "Not compatible with {arch} devices.",
+  "CheckThenUpdate": "Vérifier puis mettre à jour",
```
Comment (Nsbx) [inline main/src/assets/lang/fr_FR.json:354]: Duplicate with line 345 ?

File: main/src/assets/lang/fr_FR.json (hunk 1 above)
Comment (zhanghengxin) [reply main/src/assets/lang/fr_FR.json:344]: you are right.

**Comment by zhanghengxin** [comment]
https://github.com/IceWhaleTech/CasaOS-i18n/blob/main/README.md

Please take a look at this link. If you directly submit a pull request within the project, I will still pull the files from this repository, which would overwrite your work.

**Comment by Nsbx** [comment]
My bad I didn't see this repository, I close my PR and make the changes down there

**Pull request closed without merging**